plugins/woocommerce-admin/client/tasks/fills/experimental-import-products/importTypes.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import PageIcon from 'gridicons/dist/pages';
+import ReblogIcon from 'gridicons/dist/reblog';
+
+export const importTypes = [
+	{
+		key: 'from-csv' as const,
+		title: __( 'FROM A CSV FILE', 'woocommerce' ),
+		content: __(
+			'Import all products at once by uploading a CSV file.',
+			'woocommerce'
+		),
+		before: <PageIcon />,
+	},
+	{
+		key: 'from-cart2cart' as const,
```

Comment:
@moon0326 Looks like there is a "Learn more" button on the bottom of the`cart2cart` card. Do you plan to implement it in other PR?

![Screen Shot 2022-04-27 at 10 19 11](https://user-images.githubusercontent.com/4344253/165426376-0ff72ab4-56d7-486a-975a-2995f5ef2d10.png)

Reply:
@chihsuan Yes, I plan to implement it and other features in a follow-up PR.

I added those two cards as an example.